<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMUV2</class>
 <widget class="QWidget" name="IMUV2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="data_grid">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QLabel" name="label_acceleration_22">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_acceleration_21">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="label_temperature_11">
         <property name="text">
          <string>Temperature</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_magnetic_field_22">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_magnetic_field_23">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="label_euler_angle_roll">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLabel" name="label_euler_angle_pitch">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLabel" name="label_euler_angle_heading">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QLabel" name="label_quaternion_w">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_angular_velocity_x">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_magnetic_field_42">
         <property name="text">
          <string> µT</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="label_magnetic_field_43">
         <property name="text">
          <string> µT</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="label_angular_velocity_42">
         <property name="text">
          <string> °/s</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="label_angular_velocity_41">
         <property name="text">
          <string> °/s</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_magnetic_field_21">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_acceleration_23">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_angular_velocity_22">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_angular_velocity_23">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_angular_velocity_21">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_euler_angle_21">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_euler_angle_23">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_quaternion_22">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_euler_angle_22">
         <property name="text">
          <string>P</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="label_quaternion_21">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="label_quaternion_23">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="label_quaternion_24">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLabel" name="label_linear_acceleration_21">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLabel" name="label_linear_acceleration_22">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QLabel" name="label_linear_acceleration_23">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QLabel" name="label_gravity_vector_21">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QLabel" name="label_gravity_vector_22">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="21" column="1">
        <widget class="QLabel" name="label_gravity_vector_23">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_acceleration_42">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_acceleration_41">
         <property name="text">
          <string> m/s² </string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_magnetic_field_41">
         <property name="text">
          <string> µT</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_acceleration_43">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_acceleration_x">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label_angular_velocity_43">
         <property name="text">
          <string> °/s</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QLabel" name="label_euler_angle_41">
         <property name="text">
          <string> °</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QLabel" name="label_euler_angle_42">
         <property name="text">
          <string> °</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QLabel" name="label_euler_angle_43">
         <property name="text">
          <string> °</string>
         </property>
        </widget>
       </item>
       <item row="12" column="3">
        <widget class="QLabel" name="label_quaternion_41">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="3">
        <widget class="QLabel" name="label_quaternion_42">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="3">
        <widget class="QLabel" name="label_quaternion_43">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="15" column="3">
        <widget class="QLabel" name="label_quaternion_44">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="16" column="3">
        <widget class="QLabel" name="label_linear_acceleration_41">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="17" column="3">
        <widget class="QLabel" name="label_linear_acceleration_42">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="18" column="3">
        <widget class="QLabel" name="label_linear_acceleration_43">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="19" column="3">
        <widget class="QLabel" name="label_gravity_vector_41">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="20" column="3">
        <widget class="QLabel" name="label_gravity_vector_42">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="21" column="3">
        <widget class="QLabel" name="label_gravity_vector_43">
         <property name="text">
          <string> m/s²</string>
         </property>
        </widget>
       </item>
       <item row="22" column="3">
        <widget class="QLabel" name="label_temperature_41">
         <property name="text">
          <string> °C</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_acceleration_y">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_acceleration_z">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_magnetic_field_x">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_magnetic_field_z">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_magnetic_field_y">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_angular_velocity_z">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_angular_velocity_y">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QLabel" name="label_quaternion_x">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="QLabel" name="label_quaternion_y">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="QLabel" name="label_linear_acceleration_x">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="QLabel" name="label_quaternion_z">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="17" column="2">
        <widget class="QLabel" name="label_linear_acceleration_y">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="19" column="2">
        <widget class="QLabel" name="label_gravity_vector_x">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="18" column="2">
        <widget class="QLabel" name="label_linear_acceleration_z">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="20" column="2">
        <widget class="QLabel" name="label_gravity_vector_y">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="22" column="2">
        <widget class="QLabel" name="label_temperature">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="21" column="2">
        <widget class="QLabel" name="label_gravity_vector_z">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="0" rowspan="4">
        <widget class="QLabel" name="label_quaternion_11">
         <property name="text">
          <string>Quaternion</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" rowspan="3">
        <widget class="QLabel" name="label_euler_angle_11">
         <property name="text">
          <string>Euler Angle</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" rowspan="3">
        <widget class="QLabel" name="label_angular_velocity_11">
         <property name="text">
          <string>Angular Velocity</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" rowspan="3">
        <widget class="QLabel" name="label_magnetic_field_11">
         <property name="text">
          <string>Magnetic Field</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QLabel" name="label_acceleration_11">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Acceleration</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" rowspan="3">
        <widget class="QLabel" name="label_linear_acceleration_11">
         <property name="text">
          <string>Linear Acceleration  </string>
         </property>
        </widget>
       </item>
       <item row="19" column="0" rowspan="3">
        <widget class="QLabel" name="label_gravity_vector_11">
         <property name="text">
          <string>Gravity Vector</string>
         </property>
        </widget>
       </item>
       <item row="22" column="1">
        <widget class="QLabel" name="label_temperature_21">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_bottom">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="led_button">
             <property name="text">
              <string>Turn LEDs Off</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear_graphs">
             <property name="text">
              <string>Clear Graphs</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="save_orientation">
             <property name="text">
              <string>Save Orientation</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
